Modules/Core/Common/test/itkCompensatedSummationTest2.cxx

```diff
@@ -71,7 +71,6 @@ class CompensatedSummationTest2Associate
     void
     ThreadedExecution(const DomainType & subdomain, const itk::ThreadIdType threadId) override
     {
-      itk::CompensatedSummation<double> compensatedSum;
```

Comment:
Interestingly, the commit which "defaulted" the default-constructor of `itk::CompensatedSummation` helped the compiler (clang) to find this unused variable! 😃  At https://open.cdash.org/viewBuildError.php?type=1&buildid=8482888